web-services/client/src/main/java/datawave/webservice/result/keyword/TagCloudResponseBase.java

```diff
@@ -0,0 +1,20 @@
+package datawave.webservice.result.keyword;
+
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessOrder;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorOrder;
+import javax.xml.bind.annotation.XmlAccessorType;
+
+import datawave.webservice.result.BaseQueryResponse;
```

Comment:
After merging https://github.com/NationalSecurityAgency/datawave/pull/2815, perhaps this should go into the bast rest responses module

Reply:
Ok, cool - I will look into that. I think this needs a rebase a this point.

Reply:
Hi @ivakegg  - So, I looked into this after a rebase. The majority of the Event query response classes like `DefaultEvent` and `DefaultField` and their related base interfaces still seem to live in `datawave-ws-client`. It feels a little strange to put `DefaultTagCloud` and it's base classes in `base-rest-response` as a result because these classes largely follow the pattern of the event response classes.  I'm I misinterpreting your suggestion?

Reply:
I see what you mean.  I will have to renig on my comment.  I am honestly surprised and I wonder of those were put in datawave-ws-client because base-rest-responses was a separate module and hence hard to update.  base-rest-responses is now part of the datawave core set of modules, so perhaps we will refactor some of that down the road.  You can leave your stuff as is for now.